src/views/quickPickNode.ts

```diff
@@ -0,0 +1,13 @@
+import { QuickPickItem } from "vscode";
+
+export class QuickPickNode implements QuickPickItem {
```

Comment:
`label` and `description` is already in `QuickPickItem`. What's the purpose of `QuickPickNode`